roles/ceph-container-common/tasks/pre_requisites/prerequisites.yml

```diff
@@ -88,4 +88,13 @@
         with_pkg
   when:
     - ansible_os_family == 'RedHat'
-    - ansible_distribution_major_version == '8'
\ No newline at end of file
+    - ansible_distribution_major_version == '8'
+
+- name: ensure tmpfiles.d is present
+  lineinfile:
+    path: /usr/lib/tmpfiles.d/ceph-common.conf
+    line: "d /run/ceph 0770 ceph ceph -"
+    owner: root
```

Comment:
Why root?

Reply:
I just took the same ownership than what is done by the packaging actually

Reply:
(It shouldn't be an issue since it's 644 and only read by systemd?)